### ISSUE TITLE
`devcontainer up` is broken after upgrading from v0.26.1 to v0.27.0

### ISSUE THREAD

**Issue by lsorber** [devcontainers/cli#360]
This GitHub Actions pipeline fails on `devcontainer up` with v0.27.0: https://github.com/radix-ai/poetry-cookiecutter/actions/runs/3837027122/jobs/6531866772

If I downgrade to v0.26.1 then the same pipelines does work: https://github.com/radix-ai/poetry-cookiecutter/actions/runs/3837160421/jobs/6532150933 

The issue appears to have something to do with this output:
```
[2023-01-04T10:08:42.127Z] Start: Run: docker inspect --type image python:
[2023-01-04T10:08:42.147Z] Parsed version '' for input 'docker.io/library/python:' failed validation.
[2023-01-04T10:08:42.148Z] Error fetching image details: Could not parse image name 'python:'
[2023-01-04T10:08:42.148Z] Start: Run: docker pull python:
[2023-01-04T10:08:42.166Z] invalid reference format
[2023-01-04T10:08:42.168Z] []
[2023-01-04T10:08:42.168Z] Error response from daemon: no such image: python:: invalid reference format
```

The Dockerfile that is used in this build starts with:
```Dockerfile
ARG PYTHON_VERSION=3.8
FROM python:$PYTHON_VERSION-slim AS base
```

**Comment by ShauryaAg** [comment]
@lsorber currently, a fix that is working is changing the Dockerfile to this:

```Dockerfile
ARG PYTHON_VERSION=3.8
FROM python:${PYTHON_VERSION}-slim AS base
```

**Comment by samruddhikhandale** [comment]
Thanks @ShauryaAg for the fix. https://github.com/devcontainers/cli/pull/363 preps for the release which includes it.

**Comment by samruddhikhandale** [comment]
Closing as fixed with `v0.27.1` 🚀
